app/presenters/licences/licence-set-up.presenter.js

```diff
@@ -0,0 +1,138 @@
+'use strict'
+
+/**
+ * Formats data for the `/licences/{id}/licence-set-up` view licence set up page
+ * @module LicenceSetUpPresenter
+ */
+
+const { formatLongDate } = require('../base.presenter.js')
+
+const roles = {
+  billing: 'billing',
+  workflowEditor: 'charge_version_workflow_editor',
+  workflowReviewer: 'charge_version_workflow_reviewer'
+}
+
+/**
+ * Formats data for the `/licences/{id}/licence-set-up` view licence set up page
+ *
+ * @returns {Object} The data formatted for the view template
+ */
+function go (chargeVersions, workflows, auth, licence) {
+  return {
+    ..._authorisedLinks(auth, licence),
+    chargeInformation: _chargeInformation(chargeVersions, workflows, auth)
+  }
+}
+
+function _authorisedLinks (auth, licence) {
+  if (auth.credentials?.scope?.includes(roles.workflowEditor) && _isLessThanSixYearsOld(licence.startDate)) {
+    return {
+      setupNewCharge: `/licences/${licence.id}/charge-information/create`,
+      makeLicenceNonChargeable: `/licences/${licence.id}/charge-information/non-chargeable-reason?start=1`
+    }
+  }
+}
+
+function _chargeInformation (chargeVersions, workflows, auth) {
+  return [
+    ..._workflows(workflows, auth),
+    ..._chargeVersions(chargeVersions)
+  ]
+}
+
+function _chargeVersions (licenceSetUp) {
+  return licenceSetUp.map((chargeInformation) => {
+    return {
+      id: chargeInformation.id,
+      startDate: chargeInformation.startDate ? formatLongDate(chargeInformation.startDate) : '-',
+      endDate: chargeInformation.endDate ? formatLongDate(chargeInformation.endDate) : '-',
+      status: _status(chargeInformation.status),
+      reason: chargeInformation.changeReason?.description,
+      action: [
+        {
+          text: 'View',
+          link: `/licences/${chargeInformation.licenceId}/charge-information/${chargeInformation.id}/view`
+        }
+      ]
+    }
+  })
+}
+
+function _isLessThanSixYearsOld (date) {
+  const startDate = new Date(date)
```

Comment:
I did not see a date library in the package json ? 

Day Js will do this better - https://day.js.org/docs/en/query/is-before

Reply:
I'm sure it does. But we only need to determine this one 'thing' and I don't believe in adding dependencies unless they offer something you can't do or that would take a disproportionate amount of effort to replicate.